01/niilz_stroh/wordvalue.py

```diff
@@ -0,0 +1,28 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY, "r") as f:
+        words = f.read()
+        f.close()
```

Comment:
no need for `f.close()`, the with block (context manager) does that automatically